components/header/TopBar.tsx

```diff
@@ -0,0 +1,99 @@
+import Image from "apps/website/components/Image.tsx";
+import { Logo } from "../../components/header/Header.tsx";
+
+// Make it sure to render it on the server only. DO NOT render it on an island
+function TopBar({
+  logoBrandili,
+  logoMundi,
+  bgColor,
+  device,
+}: {
+  logoBrandili?: Logo;
+  logoMundi?: Logo;
+  device: "mobile" | "desktop" | "tablet";
+  bgColor?: "bg-red-0" | "bg-gold-1";
+}) {
+  // Mobile header
+  if (device === "mobile") {
+    return (
+      <div
+        class={`${bgColor ?? "bg-red-0"} lg:hidden flex items-center w-full`}
+      >
+        {logoBrandili && (
+          <a
+            href="/"
+            aria-label="Brandili logo"
+            className={`block px-2.5 py-1.5 rounded-tr ${
+              bgColor === "bg-red-0" ? "bg-white-0" : "opacity-50 "
+            }`}
+          >
+            <Image
+              src={logoBrandili.src}
+              alt={logoBrandili.alt}
+              width={logoBrandili.width || 100}
+              height={logoBrandili.height || 13}
+            />
+          </a>
+        )}
+        {logoMundi && (
+          <a
+            href="/mundi"
+            aria-label="Mundi logo"
+            className={`block  px-2.5 py-1.5 rounded-tl ${
+              bgColor === "bg-gold-1" ? "bg-white-0" : "opacity-50 "
+            }`}
+          >
+            <Image
+              src={logoMundi.src}
+              alt={logoMundi.alt}
+              width={logoMundi.width || 100}
+              height={logoMundi.height || 13}
+            />
+          </a>
+        )}
+      </div>
+    );
+  }
+
+  // Desktop header
+  return (
+    <div className={`${bgColor ?? "bg-red-0"} flex items-center w-full`}>
+      <div className="flex">
```

Comment:
No caso do header desktop, ele pode ser o ELSE do seu ternário para o retorno do componente, ficando dessa forma:
```suggestion
  // Desktop header
  ...
  ) : (
  conteúdo do componente de header desktop
  )
```